hack/build/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest AS build-stage0
+ARG OC_VERSION="stable"
+ENV OC_URL="https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/${OC_VERSION}"
+
+# Directory for the extracted binary
+RUN mkdir -p /out
+
+# Install the latest OC Binary from the mirror and scripts
+RUN mkdir /oc
+WORKDIR /oc
+
+# Download the checksum
+RUN curl -sSLf ${OC_URL}/sha256sum.txt -o sha256sum.txt
+
+# Download the binary tarball
+RUN /bin/bash -c "curl -sSLf -O ${OC_URL}/$(awk -v asset="openshift-client-linux" '$0~asset {print $2}' sha256sum.txt)"
+
+# Check the tarball and checksum match
+RUN sha256sum --check --ignore-missing sha256sum.txt
+RUN tar --extract --gunzip --no-same-owner --directory /out oc --file *.tar.gz
+
+# Make binaries executable
+RUN chmod -R +x /out
+
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
I think we can use minimal? https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?container-tabs=overview

Reply:
More docs: https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image